### ISSUE TITLE
Check that the Battle.net edition is supported in all platforms

### ISSUE THREAD

**Issue by timfel** [Wargus/wargus#159]
@Mistranger mentioned inconsistencies in the readmes. Right now, Linux and Windows builds are compiled with StormLib, but I haven't been able to test if they work. Also, the Mac build should be fixed to also compile correctly with StormLib.

**Comment by Mistranger** [comment]
When I have released 2.3, I've checked both Windows installer and debian package and they have extracted BNE CD properly. I've also removed every "BNE not supported" messages from the sources, kinda strange to see them again.
Current wartool output on windows: http://pastebin.com/RZQ8wthL
NSIS script should skip expansion page in case of BNE CD.

**Comment by timfel** [comment]
I imported the latest launchpad commit and took the README from the launchpad description page. The BNE unsupported stuff was in that README when we imported from launchpad.
That wartool output looks ok, I guess? I haven't touched the extraction code of Wargus at all. The last few commits are by you and mention BNE support. I'm guessing the build script is wrong somewhere if the debian package and installer aren't working?
